modules/metastorage/src/testFixtures/java/org/apache/ignite/internal/metastorage/server/SimpleInMemoryKeyValueStorage.java

```diff
@@ -70,6 +73,8 @@
  * Simple in-memory key/value storage for tests.
  */
 public class SimpleInMemoryKeyValueStorage extends AbstractKeyValueStorage {
+    private final ReadWriteLock rwLock = new ReentrantReadWriteLock();
```

Comment:
Why did you have to add this lock?

Reply:
Moved from parent class, it was there as a protected field before my changes